### PR TITLE
Don't force link gettext on macOS for travis-ci

### PR DESCRIPTION
**Pull request by ralreegorganon**
#### Summary

SUMMARY: None

#### Purpose of change

Fix the travis-ci build for macOS.

#### Describe the solution

Stop trying to force link gettext and set the variables as instructed by brew  instead.

#### Testing

This PR is the test--we'll see if it builds now.

### CODE REVIEW

**Comment by kevingranade** [comment]
git timeout on the Mac test, COME ON Travis.

File: build-scripts/build.sh [hunk 1]
```diff
@@ -15,6 +15,13 @@ function run_tests
 # We might need binaries installed via pip, so ensure that our personal bin dir is on the PATH
 export PATH=$HOME/.local/bin:$PATH
 
+# If building on macOS we need to set these so our builds can find the Homebrew version of gettext.
+if [ "$TRAVIS_OS_NAME" == "osx" ]
+then
+    set -gx LDFLAGS "-L/usr/local/opt/gettext/lib"
```
Comment (ZhilkinSerg) [inline build-scripts/build.sh:21]:
```
+set -gx LDFLAGS -L/usr/local/opt/gettext/lib

build-scripts/build.sh: line 21: set: -g: invalid option

set: usage: set [--abefhkmnptuvxBCHP] [-o option] [arg ...]
```

Comment (ralreegorganon) [reply build-scripts/build.sh]: Yup--saw that this morning. Attempt #3.

**Comment by ralreegorganon** [comment]
Closing in favor of #39569

**Pull request closed without merging**